add-on/chrome/bootstrap.js

```diff
@@ -523,13 +523,27 @@ var WindowListener = {
       },
 
       /**
+       * Set the current room token
+       *
+       * Push message parameters:
+       * - {String} roomToken  The current room that the link generator is connected to.
```

Comment:
Since we're using null as a parameter as well, please document the semantics of what it means to have a roomToken set to null that here.  <http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters>.